src/components/card/card.js

```diff
@@ -31,28 +45,52 @@ class Card extends React.PureComponent {
 
     return (
       <a
-        className="dr-ui--card color-gray-dark transition color-blue-on-hover round clip inline-block w-full unprose pb18"
+        className="dr-ui--card text-color-gray-dark transition color-blue-on-hover round clip inline-block w-full unprose pb18"
         href={link || path}
+        onMouseEnter={toggleHover}
+        onMouseLeave={toggleHover}
         {...externalLinkAttributes}
       >
         {renderedThumbnail}
-        <div>
+        <div className="transition">
           {(renderedLevel || renderedLanguage) && (
             <div className="flex mb6 txt-bold color-gray txt-s">
               {renderedLevel}
               {renderedLanguage}
             </div>
           )}
-          <div className="mb6 flex flex--center-cross">
-            {title}{' '}
+          <div className="mb6 flex txt-bold flex--center-cross">
+            <div>{title} </div>
+            <div className="w60">
+              <svg
+                className="transition"
+                style={{
+                  marginLeft: isHovered ? '15px' : '10px',
+                  opacity: isHovered ? '1' : '0'
+                }}
+                width="11"
+                height="11"
+                viewBox="0 0 11 11"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
```

Comment:
We should move this arrow icon out into a component so it is re-usable, and so it's clearer what it is in the code.